pagopa/io-app
# fix: [IOBP-773,IOBP-759] Hide dueDate if not provided in the payment details

## Short description
This PR hides the `dueDate` field if not present when getting the details of a payment.

## List of changes proposed in this pull request
- Added a check in the `WalletPaymentDetailScreen` to hide the due date field if not provided from the backend response.
- Added a check to hide the due date if the field is equal to `31/12/2099` which is a dumb value that some entities returns to us indicating that this notice code doesn't have a due date

## How to test
Start a payment with a notice code that doesn't provide a due date, you should be able to not see it rendered
